Core1Platform/BasicPhilosophy/01_General.md

```diff
@@ -0,0 +1,17 @@
+# General information
+
+The *Actindo Core1 Platform* offers a comprehensive solution for efficient digital Unified Commerce. It is an API-first and cloud-native *Digital Operations Platform (DOP)*. It provides you with an easy and fast integration of new business models in an existing IT landscape.   
+The platform brings together a high degree of automation and process integration. It offers a fast time to market as well as tremendous relief and time savings in day-to-day operations.   
```

Comment:
Dieser und nächster Satz zusammen mit dem ersten Satz (ich finde, die gehören alle in einem Absatz).